support/ci/deploy_mac_launcher.sh

```diff
@@ -48,9 +49,10 @@ export BINTRAY_PASSPHRASE=$BINTRAY_PASSPHRASE
 export TRAVIS_BUILD_NUMBER=$TRAVIS_BUILD_NUMBER
 EOF
 
-scp ${ssh_args} ${var_file} ${mac_builder}:~/tmp
+scp "${ssh_args[@]}" ${var_file} ${mac_builder}:~/tmp
 rm ${var_file}
 
 # kick off the build
-ssh ${ssh_args} ${mac_builder} \
+# shellcheck disable=2029
```

Comment:
Same as [above](https://github.com/habitat-sh/habitat/pull/4974/files#r192913698), I assume this is correct